RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -10,17 +10,17 @@ Services
   }
   ReqProxy
   {
-    Port = 9198
+    Port = 9161
```

Comment:
Hi Federico,

I have so far found no explanation in the google forum. Only as question from Ueda and your response stating that the change has been reverted (but no explanation).

Cheers,
Raja.

On 15/07/16 10:55, fstagni wrote:
> In RequestManagementSystem/ConfigTemplate.cfg <https://github.com/DIRACGrid/DIRAC/pull/3032#discussion_r70948791>:
>
>> @@ -10,17 +10,17 @@ Services
>>    }
>>    ReqProxy
>>    {
>> -    Port = 9198
>> +    Port = 9161
>
> Please check the google forum for an explanation
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/DIRACGrid/DIRAC/pull/3032/files/e53d75a34a6254a4e15f6ed2f56e72d6f73e634d#r70948791>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAwQkdX6TuvJ54VSnZUj_RmpoIL5N9syks5qV1kIgaJpZM4JKeOa>.
>